tests/regression/roles/sonic_vxlan/defaults/main.yml

```diff
@@ -89,6 +92,8 @@ tests:
         vrf_map:
           - vni: 102
             vrf: "{{vrf2}}"
+        suppress_vlan_neigh:
+          - vlan_name: Vlan6
```

Comment:
Vlan6 was previously configured as a "suppress_vlan_neigh" Vlan in test_04.

The reason for requesting the addition of another test entry in this test was to test adding another Vlan to an existing "suppress vlan neighbor" list. This Vlan will not be visible after the diff is obtained and will not cause any action. Please specify some Vlan other than 6 or 7 here.